gstd/src/common/errors.rs

```diff
@@ -115,6 +115,12 @@ impl From<CoreError> for Error {
     }
 }
 
+impl From<TryFromSliceError> for Error {
+    fn from(_: TryFromSliceError) -> Self {
+        Self::Convert(TryFromSliceError::REASON)
```

Comment:
for backwards compatibility with `gstd::Error`, but in general we could break it